updater/lib/dependabot/updater/operations/group_update_all_versions.rb

```diff
@@ -37,7 +37,7 @@ def initialize(service:, job:, dependency_snapshot:, error_handler:)
           @dependency_snapshot = dependency_snapshot
           @error_handler = error_handler
           # This is a placeholder for a real rule object obtained from config in future
-          @dependency_group = Dependabot::DependencyGroup.new(name: GROUP_NAME_PLACEHOLDER)
+          @dependency_group = Dependabot::DependencyGroup.new(name: GROUP_NAME_PLACEHOLDER, rules: [])
```

Comment:
nit: I think it would make sense to update GROUP_NAME_PLACEHOLDER here to "all-dependencies". Otherwise branch names will pop up with a `*`

Reply:
Good spot, yep, we should be consistent

Reply:
Ah, I've actually realised we don't use this name to generate the branch name as API is the one that _actually_ generates the object that gets used.

I'm going to leave this as-is for now to avoid CI churn, etc as we should yoink this placeholder out once #7075 merges